ccsds/TC-Packet.acn

```diff
@@ -38,23 +38,40 @@ TC-PacketPrimaryHeader []
     packetDataLength []
 }
 
-TC-PacketDataField []
+TC-PacketDataField <INTEGER:secondaryHeaderFlag> []
 {
-    packetSecondaryHeader [],
-    userData [align-to-next byte]
+--! packetSecondaryHeader [present-when secondaryHeaderFlag],
+    dataWithSecondaryHeader [present-when secondaryHeaderFlag==1]
+    {
+        secondaryHeader [],
+        userData <secondaryHeader.messageType-ID.serviceType-ID,
+                  secondaryHeader.messageType-ID.messageSubtype-ID,
+                  secondaryHeaderFlag> []
+    },
+    dataNoSecondaryHeader [present-when secondaryHeaderFlag==0]
+    {
+        userData []
+    }
 }
 
-TC-UserDataField <INTEGER:service, INTEGER:message> []
+TC-UserDataField <INTEGER:service, INTEGER:message, INTEGER:secondaryHeaderFlag> []
 {
-    data <service, message> [],
-    --spare [],
+    data <service, message, secondaryHeaderFlag> [],
+--! spare [],
```

Comment:
dead code? maybe comment should state something like "spare modelled using align-to"